.github/workflows/ci.yml

```
@@ -316,8 +316,8 @@ jobs:
           GIT_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Promote Artifacts in S3 and Docker Hub
-        if: github.ref == 'refs/heads/master' && env.CONTRIB_DOCKER_USR != ''
+        if: github.ref == 'refs/heads/main' && env.CONTRIB_DOCKER_USR != ''
         run: make -j2 promote
         env:
-          BRANCH_NAME: master
-          CHANNEL: master
+          BRANCH_NAME: main
+          CHANNEL: main
```

```suggestion
          CHANNEL: master
```

We are not doing channel renames with the branch name. We should do those separately.

---

Good catch Jean 🙏